### ISSUE TITLE
prebuilt wheel (jaxlib==0.1.56+cuda101) incompatible with manylinux2010

### ISSUE THREAD

**Issue by feinsteinben** [google/jax#4677]
I'm having a server with CentOS 7.6, on which there is a conda environment with python 3.8, cudatoolkit 10.1 and cudnn 7.6.5 (and all the other requirements for jax).
I'm tried to install jax with cuda support following this line (I used `pip download` to verify that all the dependencies are installed through conda, and then used `pip install --no-deps` to prevent pip from overwriting conda packages):
```bash
pip install --no-deps jax jaxlib==0.1.56+cuda101 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
The downloaded wheel file is this: [https://storage.googleapis.com/jax-releases/cuda101/jaxlib-0.1.56%2Bcuda101-cp38-none-manylinux2010_x86_64.whl](https://storage.googleapis.com/jax-releases/cuda101/jaxlib-0.1.56%2Bcuda101-cp38-none-manylinux2010_x86_64.whl)

When I try to import jax in python I get the following error (the path to my conda environment was replaced with `<env-path>`):
```
ImportError                               Traceback (most recent call last)
<ipython-input-1-cb15c4215ef7> in <module>
----> 1 import jax

<env-path>/lib/python3.8/site-packages/jax/__init__.py in <module>
     20 # flake8: noqa: F401
     21 from .config import config
---> 22 from .api import (
     23   ad,  # TODO(phawkins): update users to avoid this.
     24   argnums_partial,  # TODO(phawkins): update Haiku to not use this.

<env-path>/lib/python3.8/site-packages/jax/api.py in <module>
     35 from contextlib import contextmanager, ExitStack
     36
---> 37 from . import core
     38 from . import lib
     39 from . import linear_util as lu

<env-path>/lib/python3.8/site-packages/jax/core.py in <module>
     29 import numpy as np
     30
---> 31 from . import dtypes
     32 from .config import FLAGS, config
     33 from . import linear_util as lu

<env-path>/lib/python3.8/site-packages/jax/dtypes.py in <module>
     29 from . import util
     30 from .config import flags
---> 31 from .lib import xla_client
     32
     33 FLAGS = flags.FLAGS

<env-path>/lib/python3.8/site-packages/jax/lib/__init__.py in <module>
     51 _check_jaxlib_version()
     52
---> 53 from jaxlib import xla_client
     54 from jaxlib import lapack
     55 if version <  (0, 1, 53):

<env-path>/lib/python3.8/site-packages/jaxlib/xla_client.py in <module>
     37 # and TensorFlow may fail with duplicate protocol buffer message definitions.
     38
---> 39 from . import xla_extension as _xla
     40
     41 # Most functions are snake_case for consistency with other modules, some

ImportError: /usr/lib64/libm.so.6: version `GLIBC_2.23' not found (required by <env-path>/lib/python3.8/site-packages/jaxlib/xla_extension.so)
```

According to the error message, it seems that `xla_extension.so` depends on GLIBC_2.23 which is not compatible with `manylinux2010`. I looked into related issues and PRS and found those:
#854 GLIBC issue
#1173 jaxlib wheels aren't manylinux2010 compliant
#1178 Update Docker build to produce manylinux2010 compliant wheels.
#3555 **Make CUDA wheels manylinux2010 compliant, add CUDA 11, drop CUDA 9.2** (seems most relevant)

**Comment by xidulu** [comment]
I met the same problem on Cent OS 7 with Python 3.6.
The wheel downloaded is 
```https://storage.googleapis.com/jax-releases/cuda102/jaxlib-0.1.56%2Bcuda102-cp36-none-manylinux2010_x86_64.whl ```

**Comment by hawkinsp** [comment]
We've released new jaxlib wheels (0.1.57) which should be manylinux2010 compatible. Hope that helps!

**Comment by xidulu** [comment]
@hawkinsp 
Thanks for the latest wheel release, that solves the issue on my server.
